.golangci.yml

```diff
@@ -1,3 +1,18 @@
 run:
   deadline: 5m
   allow-parallel-runners: true
+
+linters:
+  enable:
+    - dupl
+    - errcheck
+    - exportloopref
+    - gocyclo
+    - gofmt
+    - gosimple
+    - govet
+    - ineffassign
+    - staticcheck
+    - typecheck
+    - unparam
+    - unused
```

Comment:
It looks like your IDE or something is not adding the trailing line break to the last line.
This is a usual issue for everybody, and it makes GitHub mad about that.